src/Components/Post/index.jsx

```diff
@@ -1,34 +1,35 @@
 import React from 'react';
-import Title, {SIZE_SMALL} from "../Title";
-import {POST_TEXT, POST_TITLE} from "../../strings";
+import Title from "../Title";
+import {POST_TEXT, POST_TITLE} from "../../Constants/strings";
 import Text from "../Text";
 import SocialMedia, {FACEBOOK, INSTAGRAM} from "../SocialMedia";
 import './index.css'
 import PostImage1 from './ai.jpg'
-import PostImage2 from './ai2.jpg'
 
-const Post = ({revert}) => {
+const PostDisplay = ({
+                  revert = true,
+                  socialMedia = true,
+                  post = {
+                      title: POST_TITLE, description: POST_TEXT, photo: PostImage1
+                  }
+              }) => {
+
     return (
-        !revert ?
-            <div className='postView'>
-                <img src={PostImage2} className='postImage' alt='ai'/>
-                <div className='space'/>
-                <div className='postContent'>
-                    <Title size={SIZE_SMALL} text={POST_TITLE}/>
-                    <Text text={POST_TEXT}/>
-                    <SocialMedia medias={[FACEBOOK, INSTAGRAM]}/>
-                </div>
-            </div> :
-            <div className='postView'>
-                <div className='postContent'>
-                    <Title size={SIZE_SMALL} text={POST_TITLE}/>
-                    <Text text={POST_TEXT}/>
-                    <SocialMedia medias={[FACEBOOK, INSTAGRAM]}/>
-                </div>
-                <div className='space'/>
-                <img src={PostImage1} className='postImage' alt='ai'/>
+        <div className='postView'>
+            <img src={post.photo}
+                 className={revert ? 'postImage1' : 'postImage2'}
+                 alt='ai'/>
+            <div className='postContent'>
+                <Title text={post.title}/>
+                <Text text={post.description}/>
+                {
+                    socialMedia ?
+                        <SocialMedia medias={[FACEBOOK, INSTAGRAM]}/> :
+                        null
+                }
```

Comment:
As `socialMedia` is a boolean you can use `&&`  for conditional rendering. `socialMedia && <SicialMedia/>`